Small cleanup

nickmeinhold opened the pull request:
I just pulled this out of my next PR to merge separately so the next PR contains only changes.  Just a couple of little cleanup items.

- Update to match style guide 
  - removed commented code 
  - changed ScreenHome to HomeScreen
- Add const everywhere analyzer complains

OppositeDragon reviewed (commented):
I have already changed the style for all screen names

nickmeinhold commented:
> I have already changed the style for all screen names

What changes would you like made to the PR?  

If you look at the 'Files changed' you can see that one Screen was still named the old way so I changed it - the branch is up to date I believe.

If you're not approving a PR when you review it's super helpful if you can suggest a change so we have a course of action to discuss and eventually agree on, it might seem obvious to you what the action is but maybe not to someone else and that ambiguity is where a lot of miscommunication occurs in my experience 🙂
